PL2/PL2-core/src/test/java/nl/tudelft/pl2016gr2/core/algorithms/FilterSnipsTest.java

```diff
@@ -1,51 +1,12 @@
 package nl.tudelft.pl2016gr2.core.algorithms;
 
-import nl.tudelft.pl2016gr2.model.Node;
-import nl.tudelft.pl2016gr2.model.OriginalGraph;
 
 import org.junit.Test;
 
 public class FilterSnipsTest {
 
   @Test
   public void testMakeSnip() {
-    OriginalGraph graph = new OriginalGraph();
```

Comment:
I see, Since this violating hunk of code is outdated with #52, there is no need to correct it here. We can update it with `dev` when #52 is merged.